packages/common/src/schema.graphql

```diff
@@ -1082,6 +1082,15 @@ input BatchOutboundShipmentInput {
   deleteOutboundShipments: [String!]
 }
 
+input BatchInboundShipmentInput {
+  insertOutboundShipments: [InsertInboundShipmentInput!]
+  insertInboundShipmentLines: [InsertInboundShipmentLineInput!]
+  updateInboundShipmentLines: [UpdateInboundShipmentLineInput!]
+  deleteInboundShipmentLines: [DeleteInboundShipmentLineInput!]
+  updateInboundShipments: [UpdateInboundShipmentInput!]
+  deleteInboundShipments: [String!]
+}
+
```

Comment:
This is for auto generating the type - maybe I should remove but it doesn't make any difference for the api 🤔